### PR TITLE
Duplicate question interactive mode

### PR DESCRIPTION
**Pull request by suvhotta**
Fixing the issue #195

### CODE REVIEW

**Comment by suvhotta** [comment]
@hedythedev could you please check this PR ?

**Comment by hedythedev** [comment]
The `d` looks good, but I don't think you have stripped out the duplication div:
> ### Changes required
> We should remove the text by stripping out the corresponding div:
> ```
> This question already has answers here:?
>                         ?
> 
> 
> 
> 
> Loop backwards using indices in Python? [/questions/869885/loop-backwards-using-indices-in-python]
> ?
>                                (15 answers)?
> Closed 3 years ago.
> ```

File: socli/tui.py [hunk 1]
```diff
@@ -82,16 +82,35 @@ def make_frame(self, data):
...
+                    QuestionTitle(question_title),
+                    self.question_text,
+                    QuestionStats(question_stats),
+                    urwid.Divider('-')
+                ]),
+                body=self.answer_text,
+                footer=urwid.Pile([
+                    QuestionURL(question_url),
+                    UnicodeText(
+                        u'\u2191: previous answer, \u2193: next answer, o: open in browser, \u2190: back, d : visit duplicate question, q: quit')
```
Comment (hedythedev) [inline socli/tui.py:109]:
```
d : visit duplicate question
 ^
```
extra space above, and I would think `visit duplicated question` would be more suitable?

**Comment by gautamkrishnar** [comment]
@suvhotta any update on this, just a heads up! 😄

**Comment by suvhotta** [comment]
Hey @gautamkrishnar have been a bit busy lately, will be wrapping this up by this weekend Max.

**Comment by gautamkrishnar** [comment]
@suvhotta no problem take your time 😄

**Comment by suvhotta** [comment]
@gautamkrishnar Can you check this one?

File: socli/tui.py (hunk 1 above)
Comment (hedythedev) [reply socli/tui.py:109]: Thanks @suvhotta but it seems you forgot to fix this?

**Comment by hedythedev** [comment]
@gautamkrishnar @suvhotta 
The "This question already has answers here: ..." div is stripped, but now the whole question description is turned into: "This is a duplicate question" Is that what we want? Because then the user won't be able to see the question description, just that message, and the answer(s)

image

**Comment by hedythedev** [comment]
Note I'm approving the "d" key desc msg (looks good!)

**Comment by suvhotta** [comment]
So then we need to only strip the part of  This question already has answers here:?
                        ?
and keep the rest as it is? or anything else needs to be stripped as well @hedythedev ?

**Comment by hedythedev** [comment]
As said in #195 I think we should only remove the ”closed” div (which is what you wrote yes), and keep the description as is. 

CC: @gautamkrishnar

**Comment by suvhotta** [comment]
@hedythedev Bdw can you help me with how to run this module in my local after making changes?

**Comment by gautamkrishnar** [comment]
@suvhotta first do `pip uninstall socli` if you have it installed globally. Navigate to socli folder and run `python -m socli <arguments>`

**Comment by gautamkrishnar** [comment]
@suvhotta please keep the description as same, don't change it. Also, please check why the `d : visit duplicate question` is not displayed.

**Comment by suvhotta** [comment]
@gautamkrishnar I tried the steps u mentioned to replicate the module in my local, but it is throwing an error saying the module socli doesn't exist when I try to do a python -m socli arguments

**Comment by gautamkrishnar** [comment]
@suvhotta please try using `python -m socli How to loop backwards in python` it should work if you are on the project root directory. Please make sure that there's README.md available in the same directory by doing `ls`.

**Comment by hedythedev** [comment]
Sorry about the delay but you should also install all the requirements in requirements.txt. And could you paste the error traceback here

**Comment by gautamkrishnar** [comment]
@suvhotta the latest changes are looking good, please let me know once this is ready to review.

**Comment by suvhotta** [comment]
Thanks for your help guys @gautamkrishnar  and @hedythedev . I've installed the package and am able to test it now. I've made all the necessary changes as per #195 ,please review the latest commit.

**Comment by gautamkrishnar** [comment]
@suvhotta thanks a lot for contributing.

@all-contributors please add @suvhotta for code

**Pull request merged**